Linux/audit.sh

```diff
@@ -79,10 +79,39 @@ harden() {
 }
 
 
+backup_config_dirs() {
+        # Takes 1 argument - Array of strings of paths that need to be backed up.
+
+        # Get paths that exist
+        local -n dir_arr=$1
+        dir_str=""
+
+        for path in ${dir_arr[@]};
+        do
+                [[ -d $path ]] && dir_str="$dir_str $path"
```

Comment:
Maybe add some checking to see if path exists so that way the entire function doesn't fail at the end?